ios/Scripts/generate_app_icons_from_source.py

```
@@ -0,0 +1,130 @@
+            print(f"✅ {filename} ({size}x{size})")
+            return True
+
+    except (OSError, ValueError, ImportError) as e:
+        print(f"❌ Ошибка при создании {filename}: {e}")
+        return False
+
+
+def generate_all_icons_from_source(source_path: str) -> bool:
+    """Генерирует все необходимые размеры иконок из исходного изображения"""
+    try:
+        from PIL import Image
```

Flake8 (7.2.0)

[error] 'PIL.Image' imported but unused

(F401)